src/components/Post.js

```diff
@@ -223,3 +230,51 @@ const ExpandedPostDetails = props => {
     </div>}
   </div>
 }
+
+const Followers = props => {
+  let { post, currentUser } = props
+  let { followers } = post
+
+  let onlyAuthorIsFollowing = followers.length === 1 && first(followers).id === post.user.id
+
+  let meInFollowers = (currentUser && findWhere(followers, {id: currentUser.id}))
+
+  var followersNotMe
+
+  if (meInFollowers) {
+    followersNotMe = without(followers, meInFollowers)
+  } else {
+    followersNotMe = followers
+  }
+
+  let firstTwoFollowersNotMe = followersNotMe.slice(0, 2)
+
+  if (followers.length > 0 && !onlyAuthorIsFollowing) {
+    return <div className='meta followers'>
```

Comment:
:) 
I tried, and then decided to do a direct copy from the old site for accuracy. I'm happy to try a refactor, now that I've got it working the old way